### DISCUSSION TITLE
Using FeynCalc for calculation of HQET

### DISCUSSION THREAD

**Discussion by kdteam2** [FeynCalc/feyncalc#290] [Ideas]
I am new at this and I need some example snippets to calculate the decay width calculations in heavy quark effective theory. I am having trouble in defining the covariant heavy fields which go feynman diagrams. If someone can give me reference of some help, It will be much appreciated.

**Comment by sudeepan** [comment]
@kdteam2  For HQET calculations, you can use the [FeynOnium](https://github.com/FeynCalc/feynonium) add-on for FeynCalc.
